views/blogs/index.ejs

```diff
@@ -28,5 +28,8 @@
         <% }); %>
     </div>
 </div>
+<footer>
 
```

Comment:
Avoid redundancy. Since you are repeating the code in each page, just modify the footer file.